packages/bpk-component-datatable/src/utils.js

```diff
@@ -17,14 +17,57 @@
  */
 /* @flow strict */
 
+// To maintain backwards compatibility with the old API of BpkDataTable which takes columns as children
+// The `react-table` library however expects columns as an array of objects
```

Comment:
Guess at some point we would want people to move away from these old API/usage and move to the new library?

Would it be worth us adding some console warnings to prompt people to migrate but not make it urgent?